manifests/kustomize/base/pipeline/pipeline-runner-role.yaml

```diff
@@ -78,3 +78,10 @@ rules:
   - seldondeployments
   verbs:
   - '*'
+- apiGroups:
+  - argoproj.io
```

Comment:
We'll need to update the docs to say this is required for custom service accounts used as well.